integration/ig/k8s/trace_exec_test.go

```diff
@@ -104,6 +107,12 @@ func TestTraceExec(t *testing.T) {
 				if isDockerRuntime {
 					e.Runtime.ContainerImageName = ""
 				}
+
+				if e.Comm == "sh" {
+					// Not tested because it varies depending on container runtime:
+					// - containerd: "containerd-shim"
+					e.Pcomm = ""
+				}
```

Comment:
I thought about it but since the Pcomm field is tested for other processes (`date`, `sleep`), I thought it was not a big deal if it is not tested in just one case.